frontend/src/lib/components/PropertiesTable.scss

```diff
@@ -25,6 +25,15 @@
     &:not(:first-child) {
         margin-left: 0.25rem;
     }
+
+    /*
+        generates .item-popup .property-value-type
+        only needed until feature flag collaborations-taxonomy is removed
+    */
+    @at-root .item-popup & {
+        margin-left: -1px;
+        margin-top: 0.5em;
+    }
```

Comment:
nit: I think it'd be ideal if we can avoid injecting styles into the root. Is there a way to make this selector more specific (either with a more specific selector or placing it in-line js style)

Reply:
managed to remove it entirely 👍